README.md

```diff
@@ -65,6 +67,15 @@ AZURE_TENANT_ID="your-azure-tenant-id-here"
 
 Note that the environment variables, if set, take preference over the values in a configuration file.
 
+After adjusting your ```~/.azure/credentials``` file you will need to adjust your ```.kitchen.yml``` file to leverage the azurerm driver. Use the following examples to achieve this, then check your configuration with standard kitchen commands. For example,
+
+```bash
+% kitchen list
+Instance                Driver   Provisioner  Verifier  Transport  Last Action    Last Error
+wsus-windows-2012r2     Azurerm  ChefZero     Inspec    Winrm      <Not Created>  <None>
```

Comment:
Can we update the examples to more modern windows releases? In a few years these examples are going to look super dated.

Reply:
Done.